client/src/components/stars.jsx

```diff
@@ -0,0 +1,43 @@
+import styled, { css } from 'styled-components';
+import React from 'react'
+
+const StarWrapper = styled.div`
+  margin-top: 9px;
+  margin-bottom: 9px;
+  line-height: 24px;
+  display: blocK;
+`
+const NumberOfReviews = styled.div`
+  font-family: Helvetica Neue,Helvetica,Arial,sans-serif;
+  font-size: 16px;
+  color: #666666;
+  vertical-align: baseline;
+  position: relative;
+`
+
+const Rating = styled.div`
+  float: left;
+  line-height: 24px;
+  margin-bottom: 0;
+  margin-right: 6px;
+  width: 132px;
+  height: 24px;
+  background-image: url("stars.png");
+  background-position: 0 -168px;
+  position: relative;
+`
+
+
+const StarsAndReviews = (props) => {
+
+  console.log("props from the stars side", props)
+
+  return (
+    <StarWrapper>
+      <Rating height="303" width="84" /> 
```

Comment:
switching to inline style from stylizedComponents?
Or am i not getting what's going on here (also likely)

Reply:
I switched to stylized components ( i found myself doing wayy too much css to not justify it). This inline height and width is defintely a mistake i will be fixing